Hudl.Mjolnir/Command/CommandContext.cs

```diff
@@ -132,18 +143,18 @@ internal static ICircuitBreaker GetCircuitBreaker(GroupKey key)
                     new ConfigurableValue<bool>("mjolnir.breaker." + key + ".forceTripped", DefaultBreakerForceTripped),
                     new ConfigurableValue<bool>("mjolnir.breaker." + key + ".forceFixed", DefaultBreakerForceFixed));
 
-                return new FailurePercentageCircuitBreaker(key, metrics, Stats, properties);
+                return new FailurePercentageCircuitBreaker(key, metrics, Stats, MetricEvents, properties);
```

Comment:
Looking at the default `LazyThreadSafetyMode` for `GetOrAddSafe`, this could potentially be called more than once by multiple threads. In which case we will have some dangling ConfigurableValues....I think :thought_balloon: